test/fixtures/numeral.json

```diff
@@ -180,6 +180,122 @@
       ]
     ]
   },
+  {
+    "thousandsGroupStyle": "thousand",
+    "prefix": "Skr.",
```

Comment:
This is an example of data that exhibits the bug. The numeralDecimalMark is assumed to be the default, `.`.